.gitignore

```diff
@@ -34,7 +34,6 @@ sunpy/data/sample/*.fit
 
 ### https://raw.github.com/github/gitignore/master/Global/OSX.gitignore
 
-.DS_Store
```

Comment:
Why not just import the SciPy implementation of the Savitzky-Golay filter
instead of using the scipy cookbook version?

http://docs.scipy.org/doc/scipy/reference/generated/scipy.signal.savgol_filter.html#scipy.signal.savgol_filter

On Mon, Jul 13, 2015 at 10:05 AM, dpshelio <notifications@github.com> wrote:

> In .gitignore
> <https://github.com/sunpy/sunpy/pull/1483#discussion_r34465538>:
>
> > @@ -34,7 +34,6 @@ sunpy/data/sample/*.fit
> >
> >  ### https://raw.github.com/github/gitignore/master/Global/OSX.gitignore
> >
> > -.DS_Store
>
> why is this line removed? is not that the annoying folder from OSx?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sunpy/sunpy/pull/1483/files#r34465538>.
>